addons/delivery_mondialrelay/data/neutralize.sql

```diff
@@ -0,0 +1,3 @@
+-- disable mondialrelay
+UPDATE delivery_carrier
+SET mondialrelay_brand = 'dummy';
```

Comment:
Better use the valid test credentials, no?
```suggestion
SET mondialrelay_brand = 'BDTEST  ';
```

Reply:
Ah yeah, good point since it will allow users to finish the flow of using Mondial Relay without accidentally charging themselves.

Sorry about that @EmilioPascual could you apply this change?

Reply:
Change applied. Thanks.